cli/borges/producer.go

```diff
@@ -56,6 +57,12 @@ func (c *producerCmd) jobIter(b queue.Broker) (borges.JobIter, error) {
 		if err != nil {
 			return nil, err
 		}
+
+		if c.RepublishBuried {
+			if err := q.RepublishBuried(); err != nil {
```

Comment:
I think this will fail if the queue is not already created.

Reply:
Anything we can do about that? Anyway, it's a flag you have to set, it does not make sense to set it if you know there is nothing in the buried queue.